.github/workflows/frontend-format-lint-typecheck.yml

```diff
@@ -37,4 +37,7 @@ jobs:
       run: npm --userconfig=".npmrc_ci" ci
     - name: Lint Files
       working-directory: frontend
-      run: npm run lint
\ No newline at end of file
+      run: npm run lint
+    - name: Typecheck Files
```

Comment:
🤔 dunno if it would be better, as this pipeline already breaks if there are `styling issues`...whats would be best budget wise?